applications/asset_tracker/src/main.c

```diff
@@ -1569,7 +1584,7 @@ static void set_gps_enable(const bool enable)
 	 * state is not changing.
 	 */
 	if (!data_send_enabled() ||
-	    IS_ENABLED(CONFIG_AGPS_SINGLE_CELL_ONLY) ||
+	    IS_ENABLED(CONFIG_NRF_CLOUD_CELL_POS) ||
```

Comment:
similar to previous comment... i think 
```
(!IS_ENABLED(CONFIG_NRF9160_GPS) && !IS_ENABLED(CONFIG_GPS_SIM))
```
is a better replacement

Reply:
The code schedule cell_pos as well as agps. Do not want to make significant changes here.

Reply:
previously, it exited the function when single cell **only** was enabled.
now, cell positioning being enabled should not cause this function to exit early.

perhaps it would be more readable if this check was added separately:
```
if (!IS_ENABLED(CONFIG_NRF9160_GPS) && !IS_ENABLED(CONFIG_GPS_SIM)) {
		return;
}
```
there is no need to proceed with `set_gps_enable()` if the user has not enabled GPS in the application

Reply:
May I conclude as below,
	    IS_ENABLED(CONFIG_NRF_CLOUD_CELL_POS) ||
is to be removed, and 
            if (!IS_ENABLED(CONFIG_NRF9160_GPS) && !IS_ENABLED(CONFIG_GPS_SIM)) {
		return;
            }
is to be added. Also we need to change the comment of below I think
	/* Exit early if the link is not ready or if cell-based location
	 * only is enabled or if the cloud state is defined and the local
	 * state is not changing.
	 */